### ISSUE TITLE
Homepage mobile friendly

### ISSUE THREAD

**Issue by websnacks** [digital-york/ncelp#270]
Make small fixes to show homepage better on mobile.
* Buttons should be full-width
* Featured Works and Recently Uploaded should be underneath each other
* Fix Featured Works inner text
* Fix menu - make collapsible with hamburger menu

**Comment by Jimadine** [comment]
Slightly related to this...the application.js file is 1.2MB. Is all the code in that file used? Seems like whopper that could be ripe for trimming down.

**Comment by websnacks** [comment]
Yes totally agree. These are just some fixes to make the homepage more
mobile friendly. Didn't spend much time on it.

Sebastiaan

**Comment by paluchas** [comment]
Fixed issues
- Buttons should be full-width
- Featured Works and Recently Uploaded should be underneath each other
- Fix Featured Works inner text
- Fix menu - make collapsible with hamburger menu

Remaining issues
- long URL should [brek by word according to screen size](https://stackoverflow.com/questions/18157620/responsive-solution-for-long-urls-that-exceed-the-device-width)
- pure mobile performance
